crd-convert/server-key.pem

```diff
@@ -0,0 +1,5 @@
+-----BEGIN EC PRIVATE KEY-----
```

Comment:
Same comment: I don't think we need this unless/until we're implementing a webhook

Reply:
Can we remove this?